test/parallel/test-timers-clearImmediate.js

```diff
@@ -1,18 +1,12 @@
 'use strict';
-require('../common');
+const common = require('../common');
 const assert = require('assert');
 
 const N = 3;
 let count = 0;
 function next() {
-  const immediate = setImmediate(function() {
+  const immediate = setImmediate(common.mustCall(() => {
     clearImmediate(immediate);
     ++count;
-  });
+  }, 3));
 }
-for (let i = 0; i < N; ++i)
-  next();
```

Comment:
If it doesn't complain, that means the test is passing (or possibly not even doing anything). I think you should:

1. Put the `for` loop back.
2. Remove the `count` variable.
3. Drop the second argument to `common.mustCall()` as @mscdex said.